packages/netlify-cms/src/index.js

```diff
@@ -1,7 +1,28 @@
+import createReactClass from 'create-react-class';
```

Comment:
This dependency needs to be installed in the `netlify-cms` package and removed from `netlify-cms-core`.